### ISSUE TITLE
Top of page being cut off

### ISSUE THREAD

**Issue by Aphexus** [gburca/VirtMus#5]
Looks like about half an inch of the top of a page is being cut off for some reason on some pdfs.

Here is a pdf that is having that problem.

https://ufile.io/iqwxd

It looks fine other pdf viewers.

**Comment by gburca** [comment]
Is there some kind of menu bar preventing VirtMus from utilizing the whole screen real estate? I don't see any portion of the page being cut off with your sample file.

**Comment by Aphexus** [comment]
No. It doesn't happen with other pdfs. Just some. Seems the size is being misread.  If it's working on your end though then it most likely is something wrong with my system or some compatibility issue with windows.
